carma_wm/include/carma_wm/WMListener.h

```diff
@@ -95,7 +96,7 @@ class WMListener
 
 private:
   ros::Subscriber roadway_objects_sub_;
-  std::unique_ptr<WMListenerWorker> worker_;
+  std::unique_ptr<WMListenerWorker> worker_ {new WMListenerWorker};
```

Comment:
I moved WMListenerWorker.h file into `include` folder solely to make this work. I now realize I could have included the headerfile some other way without moving it. lmk if the file should stay where it was before.

Reply:
The file should stay where it is. Because the world model is used in many nodes and contexts, only a some of the headers are exposed to limit misuse as much as possible. You can revert this and send me a link to the sonar cloud error and I'll clear with a description as to why.

Reply:
It was originally in the body, so I moved it to the initialization list. Sonarcloud then said I could move this to inclass. I have it in initialization list in my last push, so it should the scanner should show.